transport/transport-runtime/src/main/java/com/google/android/datatransport/runtime/scheduling/persistence/SQLiteEventStore.java

```diff
@@ -114,6 +114,8 @@ public PersistedEvent persist(TransportContext transportContext, EventInternal e
               // TODO(vkryachko): come up with a more sophisticated algorithm for limiting disk
               // space.
               if (isStorageAtLimit()) {
+                recordLogEventDropped(
```

Comment:
Weren't you planning to check the `transportContext` if we want to log or not?

Reply:
we could record all log events, and the log events dropped count won't be aggregated, since events all come with log_source. Besides, i don't think those extra metrics will take too much space, record them could be a bonus?
we will only use `transportContext` to determine if we want to upload.

Reply:
sgtm, let's make sure that ingestion will not fail in the face of "unknown" log sources.

Reply:
It will be ok, as long as the logsource is registered in `LogSource` at [here](google3/wireless/android/play/playlog/proto/clientanalytics.proto). afaik, any client needs to register their own logsource if they plan to send data to cct.
`UNKNOWN` logsource itself is registered, so it would be fine.
Any non-registered logsource could possible be here?